libminifi/include/CronDrivenSchedulingAgent.h

```diff
@@ -45,8 +45,6 @@ class CronDrivenSchedulingAgent : public ThreadedSchedulingAgent {
       : ThreadedSchedulingAgent(controller_service_provider, std::move(repo), std::move(flow_repo), std::move(content_repo), std::move(configuration), thread_pool) {
   }
 
-  CronDrivenSchedulingAgent(const CronDrivenSchedulingAgent& parent) = delete;
-  CronDrivenSchedulingAgent& operator=(const CronDrivenSchedulingAgent& parent) = delete;
```

Comment:
No, this is already disabled in the parent class see related conversation: https://github.com/apache/nifi-minifi-cpp/pull/1474#discussion_r1049662610

Reply:
I don't think it's a bad thing to repeat the copy/move deletion in derived classes, because it provides more information locally, but it's fine like this as well.